src/content/ui-api/kubectl-gs/template-nodepool.md

```diff
@@ -65,65 +70,89 @@ Here are the supported flags:
 - `--azure-spot-vms` - Whether to use spot VMs for this node pool (defaults to false which means not to use spot VMs).
 - `--azure-spot-vms-max-price` - Max hourly price in USD to pay for one spot VM. If the current price for the VM is exceeded, the VM is deleted. If not set, the on-demand price for the same machine size is used as limit.
 
-## Example
+## Examples
+
+{{< tabs >}}
+{{< tab id="command-examples-aws" title="AWS">}}
 
 ```nohighlight
 kubectl gs template nodepool \
   --provider aws \
+  --organization acme \
   --cluster-name a1b2c \
   --description "General purpose" \
   --availability-zones eu-central-1a \
-  --organization acme \
   --aws-instance-type m5.4xlarge
 ```
 
+{{< /tab >}}
+{{< tab id="command-examples-azure" title="Azure">}}
+
+```nohighlight
+kubectl gs template nodepool \
+  --provider azure \
+  --organization acme \
+  --cluster-name a1b2c \
+  --description "General purpose" \
+  --release 16.0.2
```

Comment:
@axbarsan In case you are fast with removing the need for this, I'll remove it here again, too